### ISSUE TITLE
in dark themes the drop down arrow can't be seen clearly

### ISSUE THREAD

**Issue by rkl3ng** [HappenApps/Quiver#1215]

**Comment by rkl3ng** [comment]
Platform: macOS Mojave 10.14

issue:
When the latest version was updated, the problem arised like this:
![image](https://user-images.githubusercontent.com/22788129/49126905-6c75a900-f300-11e8-86ee-a03e37b8e510.png)
![image](https://user-images.githubusercontent.com/22788129/49126950-8adba480-f300-11e8-878c-7595664d2af6.png)

**Comment by ylian** [comment]
Yep I noticed this in some themes. Has to do with Dark Mode. Will try to fix.
